src/styles/main.scss

```diff
@@ -10,17 +13,59 @@ body {
   font-weight: 900;
 }
 
+.game-field {
+  background: #bbada0;
+  border-spacing: 10px;
```

Comment:
border-spacing style is used but it will have no effect because it applies only to table elements. You probably want to use margin or padding.